config/show_n_tell.exs

```diff
@@ -1,4 +1,4 @@
 use Mix.Config
 
 config :show_n_tell,
-  github_org: System.get_env("GITHUB_ORG") || "decisely"
+  github_org: "decisely"
```

Comment:
Why remove this configurability? We could let other orgs use this app later...

Reply:
I will add it back.

Reply:
Actually, I think the best way is to use the mix config system to do this rather than an environment variable.